src/styles/card-block.scss

```diff
@@ -0,0 +1,116 @@
+@import "./stars.scss";
+
+@import "./variables.scss";
+
+.container {
+  max-width: $card-width;
+
+  background: $white-color;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding-top: 32px;
+  padding-bottom: 16px;
+  margin-right: $card-margin-right;
+  margin-bottom: $card-margin-bottom;
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &:nth-last-child(-n + 4) {
+    margin-bottom: 0;
+  }
+
+  &__image {
+    display: flex;
+  }
+
+  &__info {
+    padding: 0 16px;
+  }
+
+  &__heading {
+    margin-bottom: 4px;
+
+    display: flex;
+
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $dark-blue;
+  }
+  &__text-small {
+    color: $grey-color;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+
+    font-size: 700;
+
+    margin-bottom: 16px;
+
+    &-text {
+      font-weight: 400;
+      font-size: 12px;
+      color: $grey-color;
+
+      display: flex;
+      align-items: flex-end;
+    }
+
+    &-cost {
+      font-weight: 700;
+      font-size: 16px;
+      color: $dark-blue;
+    }
+
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+
+    border-radius: 5px;
+    transition: color 0.5s, border-color 0.5s, background-color 0.5s;
+    color: $white-color;
+    background: $blue-color;
+    border: 1px solid transparent;
+
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+
+    &:hover {
+      border-color: $blue-color;
+      color: $blue-color;
+      background-color: $white-color;
+    }
+  }
+
+}
+
+.image {
+  margin: 0 19px;
```

Comment:
1 - don't use margins for BEM-blocks
2 - no need to set horizontal margin/paddings to every child, just set it to parent